layouts/jobs/bis.html

```diff
@@ -21,6 +21,8 @@
         <div class="markdown max-w-none">
           {{ range .Params.bis }}
             <h2><a href="{{ .link }}">{{ .name }}</a></h2>
+            <!-- {{ $link := dict "uuid" (index (last 1 (split .link "/")) 0) }} 
+            {{ partial "components/etro.html" $link}} -->
```

Comment:
While I appreciate this, I think we should remove this from there since `<!-- -->` doesn't stop it from calling the partial(s).

We should probably also make a `/docs` directory and put a `shortcodes.md` to explain all the shortcode(s) we have, and how to use them.

Reply:
Good call! Will delete the comment.